Formula/emacs-head.rb

```diff
@@ -91,7 +98,6 @@ class EmacsHead < Formula
   def install
     args = %W[
       --disable-dependency-tracking
-      --disable-silent-rules
```

Comment:
I suppose you deleted this because the default is anyway to disable silent rules. right?